apps/docs/studio/documents/component.ts

```diff
@@ -25,5 +25,39 @@ export default defineType({
       title: 'Content',
       type: 'content',
     }),
+    defineField({
+      name: 'resourceLinks',
+      title: 'Resource links',
+      description:
+        'Add links to GitHub or Figma here (only GitHub and Figma is supported)',
+      type: 'array',
+      of: [
+        {
+          type: 'object',
+          name: 'resourceLink',
+          title: 'Resource link',
+          fields: [
+            {
+              name: 'linkType',
+              type: 'string',
+              title: 'Link type',
+              options: {
+                list: [
+                  { title: 'Figma', value: 'figma' },
+                  { title: 'React', value: 'react' },
```

Comment:
Jeg vet vi har sakset denne rett av Spor, men når jeg ser den nå så føles det feil ut med React her for meg. Kan Github være bedre, siden vi tross alt skal lenke til Github? Det er det som kommer til å stå i lenketeksten tross alt 😅

Reply:
Fikset det selv siden Gabriel er på ferie.